sample-applications/blazing-coffee/BlazingCoffee/Client/Shared/Layouts/MainLayout.razor

```diff
@@ -38,73 +34,29 @@
             @Body
         </DrawerContent>
     </TelerikDrawer>
-    <TelerikWindow @bind-Visible="SettingsExpanded" Modal="true">
-        <WindowTitle>
-            @L["Settings"]
-        </WindowTitle>
-        <WindowActions>
-            <WindowAction Name="Close" />
-        </WindowActions>
-        <WindowContent>
-            <div class="k-widget k-form">
-                <div class="k-form-field">
-                    <label class="k-label k-form-label" for="theme">
-                        @L["SelectTheme"]
-                    </label>
-                    <div class="k-form-field-wrap">
-                        <ThemeChooser></ThemeChooser>
-                    </div>
-                </div>
-                <div class="k-form-field">
-                    <label class="k-label k-form-label" for="language">
-                        @L["LanguageChooser_SelectLanguage"]
-                    </label>
-                    <div class="k-form-field-wrap">
-                        <CultureChooser />
-                    </div>
-                </div>
-            </div>
-        </WindowContent>
-    </TelerikWindow>
-    <TelerikWindow @bind-Visible="DocsExpanded" Modal="true" Width="70vw">
-        <WindowTitle>
-            @L["MainLayout_Documentation"]: @Layout.DocsTitle
-        </WindowTitle>
-        <WindowActions>
-            <WindowAction Name="Close" />
-        </WindowActions>
-        <WindowContent>
-            <div class="docs-wrapper">
-                @((MarkupString)Docs)
-            </div>
-        </WindowContent>
-    </TelerikWindow>
 </TelerikRootComponent>
 
+<TelerikMediaQuery Media="(max-width: 768px)" OnChange="((changed) => WindowWidthSmall = changed)"></TelerikMediaQuery>
 @code {
-
     TelerikDrawer<DrawerItem> MenuDrawer { get; set; }
 
     bool Expanded { get; set; }
     bool SettingsExpanded { get; set; }
     bool DocsExpanded { get; set; }
+    bool WindowWidthSmall {get; set; }
     string Docs { get; set; }
 
     [CascadingParameter] public CoffeeComponentBase DocumentedComponent { get; set; }
 
     IEnumerable<DrawerItem> Data =>
     new List<DrawerItem>
```

Comment:
```suggestion
    IEnumerable<DrawerItem> Data { get; set;} = new List<DrawerItem>
```

Reply:
A lambda expression is needed in order to use the static localizer